src/hidimstat/conditional_permutation_importance.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
```

Comment:
there is already a `cpi` module. We cannot have both.

Reply:
This module replaces `cpi.py`. Do you agree with this more explicit naming? (also ping @lionelkusch )

Reply:
Yes, I saw that afterwards. Happy with more explicit naming.

Reply:
I opened the issue #39 on this topic. There was not, for the moment, a consensus.